driver-core/src/main/com/mongodb/operation/MapReduceToCollectionOperation.java

```diff
@@ -453,7 +476,7 @@ public void executeAsync(final AsyncWriteBinding binding, final SingleResultCall
 
     private CommandReadOperation<BsonDocument> createExplainableOperation(final ExplainVerbosity explainVerbosity) {
         return new CommandReadOperation<BsonDocument>(namespace.getDatabaseName(),
-                                                      ExplainHelper.asExplainCommand(getCommand(), explainVerbosity),
+                                                      ExplainHelper.asExplainCommand(getCommand(null), explainVerbosity),
```

Comment:
Acknowledged.  

It can be done as in com.mongodb.operation.FindOperation#asExplainableOperation, but probably not worth it.  bypassDocumentValidation won't change the explain plan for the query portion of the map reduce.